### PR TITLE
fix: text color related issue

### PR DESCRIPTION
**Pull request by harshvardhan-212**
## Description

Please do not leave this blank 

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build

## Have you tested locally?

- [ ] 👍 yes
- [ ] 🙅 no, because I am lazy

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md

### CODE REVIEW

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: packages/@sparrow-library/src/icons/ArrowForward.svelte [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<script>
+  export let height = "19px";
+  export let width = "19px";
+  export let color = "grey";
+</script>
```
Comment (itsmdasifraza) [inline packages/@sparrow-library/src/icons/ArrowForward.svelte:5]: make it bydefault 24px

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Comment by pratikshinde590** [comment]
Please attach the issue link

File: packages/@sparrow-teams/src/compopnents/rows/Rows.svelte [hunk 2]
```diff
@@ -66,6 +65,7 @@
           },
           displayText: "Delete Workspace",
           disabled: false,
+          textColor: "var(--text-danger-200)",
         },
```
Comment (itsmdasifraza) [inline packages/@sparrow-teams/src/compopnents/rows/Rows.svelte:69]: not required

**Comment by pratikshinde590** [comment]
Please attach the issue link

**Pull request merged**